### PR TITLE
[BUG FIX] Fix erniesage tokenize function and change the default value of cls_token_id in model.

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

Bug fixes
### PR changes

Models
### Description

Fix erniesage tokenize function and change the default value of cls_token_id in model.

### CODE REVIEW

File: examples/text_graph/erniesage/models/conv.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ def __init__(self,
                  input_size,
                  hidden_size,
                  learning_rate,
+                 cls_token_id=int(1),
```
Comment (wawltor) [inline examples/text_graph/erniesage/models/conv.py]: 这里为啥专门转一下int

Comment (ZHUI) [reply examples/text_graph/erniesage/models/conv.py]: 强调一下类型是int，实际没啥必要

Comment (ZHUI) [reply examples/text_graph/erniesage/models/conv.py]: done

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**